src/redux/Auth/auth-reducer.js

```diff
@@ -0,0 +1,49 @@
+import * as actionTypes from "../shopping-types";
+
+export const authReducer = (state = { data: null }, action) => {
+    switch (action.type) {
+        case actionTypes.REGISTER_REQUEST:
+            return {
+                loading: true,
+            };
```

Comment:
If default state keys and return keys should be same
in this case 
```suggestion
            return {
                ...state,
                loading: true,
            };
```